Instagram/Views/PostCell.h

```diff
@@ -0,0 +1,24 @@
+//
+//  PostCell.h
+//  Instagram
+//
+//  Created by Sarah Wang on 7/7/21.
+//
+
+#import <UIKit/UIKit.h>
+#import "Post.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PostCell : UITableViewCell
+
+@property (strong, nonatomic) Post *post;
```

Comment:
This is oversharing imo. Think about how you could pass less data through the `refreshData` method.